pcdsdevices/pim.py

```diff
@@ -559,17 +559,48 @@ class IM2K0(LCLS2ImagerBase):
     # Nothing else! No power meter, no zoom/focus, no filter wheel...
 
 
-class IM3L0(PPM):
+class K2700(BaseInterface, Device):
+    """
+    Keithley 2700 digital multimeter.
+
+    Currently supports reading voltage and current, direct and
+    alternating, but can be extended to measure other quantities
+    (resistance, temperature), have configurable range and integration
+    time, and allow for remote control of a K2700.
+    """
+    idn = Cpt(EpicsSignalRO, ":Identity", kind="normal")
+    reading = Cpt(EpicsSignalRO, ":Reading", kind="normal")
+    dcv = Cpt(EpicsSignalRO, ":GetDCV", kind="normal")
```

Comment:
```suggestion
    dcv = Cpt(EpicsSignalRO, ":GetDCV", kind="normal", doc="Some description here")
```
Would you mind adding some descriptions to these? While I know what IDN/DCV/ACV/DCI/ACI are, others might not. What goes in "Reading" isn't obvious to me, though